src/main/java/com/alrex/parcool/common/action/impl/WallJump.java

```diff
@@ -117,9 +117,9 @@ public void onStart(PlayerEntity player, Parkourability parkourability) {
 
 	@Override
 	public void onStartInLocalClient(PlayerEntity player, Parkourability parkourability, IStamina stamina, ByteBuffer startData) {
-		double speedScale = player.getBbWidth() / 0.6;
+		double speedScale = 0.6 / player.getBbWidth();
```

Comment:
I see. But I think this is unnatural. ParCool is originally created without knowledge of Pehkui so coding assuming existence of Pehkui is bad. If we had to change this like it, I think we rather change it to constant value.

Reply:
ahh i see what you are saying! to use constants we'll have to rewrite the code so that we add to the players current motion data rather than setting it; that way we won't have to manage the motion data ourselves and any mods that change the motion data can do so without having any weird interactions with this mod. i spent tonight figuring out the wall jump code, i'll see what i can do about the vault tomorrow.